docs/config_options.md

```diff
@@ -34,6 +34,8 @@ This is a C header file that is one of the first things included, and will persi
 ## `config.h` Options
 
 ### Hardware Options
+====================
```

Comment:
These are conflicting styles. The `###` indicates a level 3 header, while the `====` underneath indicates a level 1 header.

Reply:
Not a problem. Will change.

Just really want a good way to make sure these headers are LINKABLE.  And easily so.  
Because apparently, no link comes up for these... and it would make things SOO much easier.